package_test.go

```diff
@@ -340,7 +374,7 @@ const exampleCollectionsListResponse_1collection string = `
   }
 }`
 
-const exampleDocumentsCreateResponse_1documents string = `{
+const exampleDocumentsResponse_1documents string = `{
```

Comment:
I guess the naming (`_1documents`) doesn't make sense.
An "document response" contains always just one document.

What about just using `exampleDocumentResponse`?